src/index.html

```diff
@@ -6,9 +6,52 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+
+      <a href="#" class="imageLink">
+        <img class = "start" src="./images/logo.png"
+         alt="Moyo">
+      </a>
+
+      <nav>
+        <ul class="mainItem">
+          <li class="menuItem">
+            <a class="is-active" href="#">Apple</a>
+          </li>
+          <li class="menuItem menuItemMargin">
+            <a class="is-active" href="#">Sumsung</a>
+          </li>
```

Comment:
remove class `menuItemMargin`
add margin to all menuItem
and use :first-child pseudoclass to remove this margin for first item